extensions/amp-consent/0.1/cmps.js

```diff
@@ -71,6 +71,11 @@ CMP_CONFIG['SourcePoint'] = {
   'promptUISrc': 'https://amp.pm.sourcepoint.mgr.consensu.org/',
 };
 
+CMP_CONFIG['opencmp'] = {
+  'consentInstanceId': 'opencmp',
+  'checkConsentHref': 'https://amp.opencmp.net/consent/check',
+  'promptUISrc': 'https://cmp.opencmp.net/tcf-v2/amp/cmp.html',
+
```

Comment:
};

Reply:
Don't be sad :D
I set up content for that url, which I changed to https://cdn.opencmp.net/tcf-v2/amp/cmp.html in cmps.js.